test/integration/targets/nxos_nxapi/tasks/nxapi.yaml

```diff
@@ -21,13 +21,22 @@
   set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
 
 - name: run test cases (connection=httpapi)
-  include: "{{ test_case_to_run }} ansible_connection=httpapi connection={{ nxapi }}"
+  include: "{{ test_case_to_run }}"
+  vars:
+    ansible_connection: httpapi
+    ansible_httpapi_use_ssl: no
+    ansible_httpapi_port: 80
```

Comment:
The port cannot be hardcoded in the test. We use different port on our CI. this needs to be group_vars.

Reply:
@trishnaguha What if we create the following two variables under group_vars

1. http_port
2. https_port

And then just reference them here?  Would that work?

```yaml
  vars:
    ansible_connection: httpapi
    ansible_httpapi_use_ssl: no
    ansible_httpapi_port: "{{ http_port }}"
```

Same for the https case below

Reply:
as discussed on the call, we don't need this port as this is a default port

Reply:
In DCI we are now setting:
```yaml
# See https://github.com/ansible/ansible/pull/43651
nxapi_ssl:
  host: "{{ ansible_ssh_host }}"
  username: admin
  password: admin
  transport: nxapi
  # DCI uses default 443, so no need to specify
  use_ssl: yes
  validate_certs: False
```

Reply:
👍